### ISSUE TITLE
Documentation gives SSL Error

### ISSUE THREAD

**Issue by bernt-matthias** [OpenMS/OpenMS#6609]
For example http://www.openms.de/doxygen/release/2.8.0/html/UTILS_MetaProSIP.html gives me `SSL_ERROR_RX_RECORD_TOO_LONG`

**Comment by tjeerdijk** [comment]
Yeah, that is still on my plate. The speed of openms.de has been resolved and Knut has (almost) handed over control of openms.de to Oliver. Now I can start pestering Knut about access to the old web site so I can recover the old redirects and transfer them to the new web site. Sorry that this takes so long ....
